docs/semgrep-app/user-management.md

```diff
@@ -1,13 +1,41 @@
 ---
-slug: role-based-access-control
+slug: user-management 
 append_help_link: true
-title: Role-based access control
-description: "This page informs users of role-based access control in Semgrep App and teaches users about roles and role management."
+title: Managing users and roles 
+description: "Learn about roles, user management, and how to implement role-based access control in Semgrep App."
 ---
 
 import MoreHelp from "/src/components/MoreHelp"
 
-# Controlling access through roles
+
+# Managing users and roles
+
+Accounts enable you to manage access to Semgrep resources, such as scans and findings, with varying levels of collaboration and visibility.
+
+Semgrep App has two types of accounts:
+
+<dl>
+    <dt>Personal account</dt>
+    <dd>Every person who signs into Semgrep App is first signed in to a <strong>personal account</strong>. In a personal account, your findings, projects, private rules, and scans are visible only to you.</dd>
+    <dt>Organization account</dt>
+    <dd>To collaborate with others, create an <strong>organization account</strong>. An organization account in Semgrep App requires an existing organization from your source code management (SCM) tool. Creating this type of account integrates Semgrep App into your organization. A user can have many organizations, provided that they are a member of that organization in their SCM.</dd>
+</dl>
+
+Users from the same organization are able to sign into Semgrep App with their SCM's credentials and are automatically added to the organization account. Semgrep App is also able to detect the organization's repositories for scanning.
+
+In organization accounts, collaborators can collectively manage Semgrep App. By default, users are able to: 
+
+* Add projects to scan
+* View and triage findings
+* Determine what rules to run and actions
+* Manage tokens and other settings
+* View an organization's private rules
```

Comment:
(Semi blocking, worth considering.)
Write full stops at the end of all list items. Unless the items in the list are just single words use full stops. These list items you included are still sentences.
See example in https://developers.google.com/style/lists?hl=en:
<img width="834" alt="Screenshot 2022-06-09 at 12 31 24" src="https://user-images.githubusercontent.com/26669307/172827534-ac90af13-2e64-4fee-82da-25e2311abd07.png">